rest/model/build.go

```diff
@@ -46,6 +46,28 @@ type APIBuild struct {
 	ActualMakespan    APIDuration          `json:"actual_makespan_ms"`
 	Origin            *string              `json:"origin"`
 	StatusCounts      task.TaskStatusCount `json:"status_counts,omitempty"`
+	DefinitionInfo    DefinitionInfo       `json:"definition_info"`
+}
+
+type DefinitionInfo struct {
+	CronBatchTime *string `json:"cron,omitempty"`
+	BatchTime     *int    `json:"batchtime,omitempty"`
+}
+
+// PopulateDefinitionInfo adds cron/batchtime for the variant, if applicable. Not supported for matrices.
+// Only adds cron if populated, to be consistent with batchtime.
+func (apiBuild *APIBuild) PopulateDefinitionInfo(id, variant string) {
+	pp, err := model.ParserProjectFindOne(model.ParserProjectById(id).WithFields(model.ParserProjectBuildVariantsKey))
+	if err == nil {
+		for _, bv := range pp.BuildVariants {
+			if bv.Name == variant {
+				if bv.CronBatchTime != "" {
+					apiBuild.DefinitionInfo.CronBatchTime = utility.ToStringPtr(bv.CronBatchTime)
+				}
+				apiBuild.DefinitionInfo.BatchTime = bv.BatchTime
+			}
```

Comment:
should we error or warn if the bv specified is not found?

Reply:
I decided to skip this since it shouldn't happen and shouldn't affect the route (also since this isn't supported for matrices we can't error since it would mean the route is unusable in those cases)